src/form-builder/components/FormBuilder.jsx

```diff
@@ -68,6 +74,66 @@ export default class FormBuilder extends Component {
     }, commonConstants.toastTimeout);
   }
 
+  getValidJsonFileNames(fileNames) {
```

Comment:
as this method checks for FileFormat can we change the name to something like checkValidFileFormat

Reply:
This actually checks for file type and returns list of names of valid json files.